core/build.gradle.kts

```diff
@@ -26,7 +26,7 @@ val javadocConfig by configurations.creating {
 
 dependencies {
     // Use JUnit test framework.
-    testImplementation(libs.junit)
```

Comment:
Doesn't that mean that we have to have JUnit in our production environment?

Reply:
I would say no, because this is a `testImplementation` dependency, so it is needed only by tests, which are not packaged, published and reused in the production environment

Reply:
Right, sorry, I did not notice that the comment was associated with the old version of the code and not the new one.

So, yes, you are right. We are making junit a dependency of the library as we use assert methods in this utility.

Let me see if we can avoid doing that.